unison-src/new-runtime-transcripts/tls.output.md

```diff
@@ -130,7 +130,7 @@ test> match (decodeCert (toUtf8 self_signed_cert_pem) with
   
     ⍟ These new definitions are ok to `add`:
     
-    test.ckc3ihvvem (Unison bug, unknown term)
+    test.faj5ui4d7r (Unison bug, unknown term)
```

Comment:
For an unnamed test watch, it just makes up a GUID for the test name. I don't think we should rely on that in the transcripts, should use named tests. I'm guessing this difference is just due to the GUID generation not being sufficiently predictable, so I wouldn't worry about it. (I would fix the transcript to use a named test though)